web/app/components/Exchange/OrderBook.jsx

```diff
@@ -140,6 +140,14 @@ class OrderBook extends React.Component {
             if (this.refs.bidTransition) {
                 this.refs.bidTransition.resetAnimation();
             }
+
+            if (this.refs.vert_bids) this.refs.vert_bids.scrollTop = 0;
+        }
+
+        if (nextProps.combinedAsks !== this.props.combinedAsks || nextProps.combinedBids !== this.props.combinedBids) {
```

Comment:
These are regular mutable arrays, so this if conditions will always evaluate to true. If you need to check if they're different you can use the `are_equal_shallow` method from `lib/common/utils.js`. If you change that I'll test it and see if it works.